<launch>

	<node pkg="rviz" type="rviz" name="rviz" respawn="true" args="-d $(find pro_pilot_park)/rviz_config/conf.rviz" />

	<node pkg="tf" type="static_transform_publisher" name="base2laser" args="0.15 0 0 0 0 0 base_link laser 100"/>

	<include file="$(find hokuyo_node)/hokuyo_test.launch"/>

	<!-- <param name="use_sim_time" value="true"/> -->

	<!-- <arg name="bag" default="$(find pro_pilot_park)/bag/scan_garage.bag"/> -->
	<!-- <node pkg="rosbag" type="play" name="player" output="screen" args="&#45;&#45;clock &#45;r 1.0 $(arg bag)"/> -->

  <node pkg="pro_pilot_park" type="pro_pilot_park_node" name="pro_pilot_park" output="screen">
    <param name="angle_threshold" value="0.15"/>
    <param name="dist_threshold" value="0.05"/>
    <!-- <param name="garage_length_tolerance" value="0.3"/> -->
    <param name="garage_angle_tolerance" value="0.34"/>
  </node> 

</launch>
